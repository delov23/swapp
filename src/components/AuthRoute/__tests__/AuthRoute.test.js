import React from 'react';
import { shallow } from 'enzyme';

import { AuthRoute } from '../';

describe('<AuthRoute />', () => {
  it('should match snapshot', () => {
    expect(shallow(<AuthRoute />)).toMatchSnapshot();
  });
});
