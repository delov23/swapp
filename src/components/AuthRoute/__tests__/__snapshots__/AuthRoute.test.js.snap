// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthRoute /> should match snapshot 1`] = `
<PermissionRoute
  permission="authenticated"
/>
`;
